### ISSUE TITLE
Turbolinks?

### ISSUE THREAD

**Issue by chevinbrown** [estum/growlyflash#37]
It looks like turbolinks interferes with the operation of this gem. I haven't dug in yet. Is there a fix for Rails5/Turbolinks3?

**Comment by yurijmi** [comment]
Yeah, I can confirm that. There is definitely some interference with Turbolinks 5.0.0

**Comment by kikikiblue** [comment]
It still doesn't show flash message with turbolinks 5.0.1 (rails 5.1.4) when I use 'redirect_to'
but it shows flash message if I use 'data: {turbolinks: false}' to link_to method.
Anybody have some hint?

**Comment by chevinbrown** [comment]
Seems like this gem isn't actively maintained. I recommend you take a look at the [unobtrusive flash gem](https://github.com/leonid-shevtsov/unobtrusive_flash)

**Comment by estum** [comment]
@kikikiblue try to use [ActionDispatch::Flash::FlashHash#keep](http://api.rubyonrails.org/classes/ActionDispatch/Flash/FlashHash.html#method-i-keep) method.

**Comment by kikikiblue** [comment]
@estum I tried this code. but it does not show the notice flash message either.

```
flash.keep
return redirect_back(fallback_location: root_path), flash: {notice: 'Email sent'}
```

what is the right usage?

**Comment by kikikiblue** [comment]
I was stupid. :-p
```
flash[:notice] = 'Email sent'
flash.keep
return redirect_back(fallback_location: root_path)
```
It works with this code. Thanks @estum
